### PR TITLE
Using ruby json for health rack

### PR DESCRIPTION
**Pull request by mkristian**
for #11

### CODE REVIEW

File: leafy-rack/lib/leafy/rack/health.rb [hunk 1]
```diff
@@ -5,16 +5,15 @@ module Leafy
   module Rack
     class Health
 
-      WRITER = ::Leafy::Json::HealthWriter.new
-
       def self.response( health, env )
         data = health.run_health_checks
         is_healthy = data.values.all? { |r| r.healthy? }
+        json = env[ 'QUERY_STRING' ] == 'pretty' ? JSON.pretty_generate( data.to_hash ) : data.to_hash.to_json
```
Comment (rtyler) [inline leafy-rack/lib/leafy/rack/health.rb]: Since this is using the `json` module, I believe a `require 'json'` should be at the top

Comment (rtyler) [inline leafy-rack/lib/leafy/rack/health.rb]: I also really don't like long-winded ternary conditionals like this, there doesn't seem to be a good reason to compact this statement so much

Comment (mkristian) [inline leafy-rack/lib/leafy/rack/health.rb]: ```require 'json'``` is not really nice if you want to use mutil-json gem. I could do something like
```
begin
require 'mutil_json'
rescue LoadError
require 'json'
end
```
?

Comment (rtyler) [inline leafy-rack/lib/leafy/rack/health.rb]: Do people really still use multi-json? I thought that died when `json` become more standardized in 1.9.x.

How about a middle ground? if `const_defined?(:JSON)` fails, then try to `require 'json'`?

Comment (mkristian) [inline leafy-rack/lib/leafy/rack/health.rb]: middle ground is great :)

**Comment by rtyler** [comment]
This PR wasn't for master branch, I'm confused now

**Comment by mkristian** [comment]
my last PR all depended on each other  so merging them one by one without conflicts was OK.

sorry should have mentioned it somewhere.

**Pull request merged**